api/specs/webserver/openapi-projects-ports.yaml

```diff
@@ -72,18 +72,62 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Envelope_dict_uuid.UUID__simcore_service_webserver.projects.projects_ports_handlers.ProjectPortGet__'
+                $ref: '#/components/schemas/Envelope_dict_uuid.UUID__simcore_service_webserver.projects.projects_ports_handlers.ProjectOutputGet__'
+  /projects/{project_id}/metadata/ports:
+    get:
+      tags:
+      - project
+      summary: List Project Metadata Ports
+      description: New in version *0.12*
+      operationId: list_project_metadata_ports
+      parameters:
+      - required: true
+        schema:
+          title: Project Id
+          type: string
+          format: uuid
+        name: project_id
+        in: path
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Envelope_list_simcore_service_webserver.projects.projects_ports_handlers.ProjectMetadataPortGet__'
```

Comment:
ProjectMetadataPort**s**Get ?

Reply:
not sure I get it. is that one port or multiple port**s** ?

Reply:
yes! My fault

Reply:
the name is auto-generated by the code in ``openapi_projects_ports.py``
The response model is ``response_model=Envelope[list[ProjectMetadataPortGet]]``